client/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "client",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "7.0.0",
+    "babel-jest": "24.5.0",
+    "babel-loader": "8.0.5",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
+    "css-loader": "0.28.11",
+    "enzyme": "3.9.0",
+    "extract-text-webpack-plugin": "4.0.0-beta.0",
+    "html-webpack-plugin": "3.2.0",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "24.5.0",
+    "less": "3.0.1",
+    "less-loader": "4.1.0",
+    "react": "16.8.4",
+    "react-dom": "16.8.4",
+    "react-redux": "6.0.1",
+    "react-router": "4.3.1",
```

Comment:
are you sure that these are `dev dependencies`?
please move to `dependencies`
`redux` is also not dev dep.